tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -605,25 +582,16 @@ class StorageManager {
   /** Object that handles array consolidation. */
   Consolidator* consolidator_;
 
+  /** Stores filelocks for arrays. */
+  std::unordered_map<std::string, filelock_t> filelocks_;
```

Comment:
This only stores exclusive locks?

Reply:
Yes, good catch. I will change the name to `xfilelocks_`.